src/main.cpp

```diff
@@ -450,6 +573,40 @@ class gfxPipelineHandler {
         UpdateWindow(hWND);
         SetFocus(hWND);
     }
+
+    static void Install() {
+        InstallPatch("Enables pointer in windowed mode.", { 0x90, 0x90 }, {
+            0x4F136E,
+        });
+
+        if (!datArgParser::Get("16bit")) {
```

Comment:
The -cdepth and -zdepth args can also be used to do this, but it doesn't matter too much either way if it's kept in.